TIPS_and_FAQ.md

```diff
@@ -2,6 +2,47 @@
 
 ### As a user, how can I...
 
+#### Embedded Node.js
```

Comment:
Please either remove it, or move it to a `### As an Eclipse plugin developer, how can I... #### Reuse embedded Node.js?` and link to the org.eclipse.wildwebdeveloper.embedder.node/README.md

Reply:
Replaced this with  `As an Eclipse plugin developer...` section and a  link to [reuse the Node.js Embedder](https://github.com/eclipse/wildwebdeveloper/blob/master/org.eclipse.wildwebdeveloper.embedder.node/README.md)  added to the bottom.